pkg/filter/network/grpc/var.go

```diff
@@ -22,6 +22,10 @@ import (
 	"mosn.io/pkg/variable"
 )
 
+const (
```

Comment:
maybe it should be the top this file after package (reorder), and named as NetworkUnix and NetworkTcp (var name)

Otherwise LGTM

Reply:
Sorry mymistake. Ignore that

Reply:
> Sorry mymistake. Ignore the first, please rename variables

1. I don't want to set the two variables as public here
2. Maybe unixNetwork has better semantic information than NetworkUnix?

Reply:
It's ok that not exported, but those with the same intent preferably have the same prefix, like in C API they both have `AF_` prefix `syscall.AF_UNIX` and `syscall.AF_INET`